src/lib/core/components/visualizations/VisualizationsContainer.tsx

```diff
@@ -32,6 +39,18 @@ import { GeoConfig } from '../../types/geoConfig';
 
 const cx = makeClassNameHelper('VisualizationsContainer');
 
+// Doesn't seem to work
```

Comment:
Do you mean the TS typing doesn't work?

Reply:
The typing works fine. This component simply doesn't fix the problem I wanted it to!